python/pyspark/sql/functions.pyi

```diff
@@ -174,6 +174,8 @@ def bin(col: ColumnOrName) -> Column: ...
 def hex(col: ColumnOrName) -> Column: ...
 def unhex(col: ColumnOrName) -> Column: ...
 def length(col: ColumnOrName) -> Column: ...
+def octet_length(col: ColumnOrName) -> Column: ...
+def bit_length(col: ColumnOrName) -> Column: ...
```

Comment:
Oh we should also add it to PySpark docs - python/docs/source/reference/pyspark.sql.rst